lib/ruby_lsp/requests/support/rubocop_diagnostic.rb

```diff
@@ -19,6 +19,12 @@ class RuboCopDiagnostic
           T::Hash[Symbol, Integer],
         )
 
+        # Cache cops to attach URLs to diagnostics. Only built-in cops for now.
+        COP_TO_DOC_URL = T.let(
+          RuboCop::Cop::Registry.global.to_h,
+          T::Hash[String, [T.class_of(RuboCop::Cop::Base)]],
```

Comment:
Are the values in the hash always an array with a single element? Or is it just a regular array? If it's the latter, then the type should be this. Otherwise, just ignore.

```suggestion
          T::Hash[String, T::Array[T.class_of(RuboCop::Cop::Base)]],
```